stardis/radiation_field/opacities/opacities_solvers/broadening.py

```diff
@@ -21,6 +23,10 @@
 VACUUM_ELECTRIC_PERMITTIVITY = 1.0 / (4.0 * PI)
```

Comment:
Possibly this is not "vacuum electric permittivity". This looks like a constant.

Reply:
I agree that this is weird and I can look into it, but it's not really part of the PR and is leftover from something I can't take credit for.